src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -478,6 +527,9 @@ export default {
 
             if (viewLargeAction && viewLargeAction.appliesTo(this.objectPath, this.currentView)) {
```

Comment:
I realize you didn't write this, but I'm just wondering why we're checking `appliesTo` here. It's guaranteed to apply to the given `objectPath` and `currentView` because they were passed into `getActionsCollection`, which uses `appliesTo` to populate the collection from all available actions. I think we can get rid of this check.